global.json

```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "5.0.100",
+    "version": "6.0.100-alpha.1.21070.6",
     "allowPrerelease": true,
     "rollForward": "major"
   },
   "tools": {
-    "dotnet": "5.0.100"
+    "dotnet": "6.0.100-alpha.1.21070.6"
```

Comment:
You don't intent do commit the SDK update, right?

Reply:
I didn't intend to commit this PR.  It was more of an experiment to see if the .NET 6 SDK would work (especially on native Apple Silicon because NET 5 doesn't support Apple Silicon native.)

@jkoritzinsky has suggested we might commit something like this to solve the crossgen2 issues @trylek is working on.